plugins/common/opcua/input/input_client.go

```diff
@@ -57,6 +58,7 @@ type NodeSettings struct {
 	TagsSlice        [][]string           `toml:"tags" deprecated:"1.25.0;use 'default_tags' instead"`
 	DefaultTags      map[string]string    `toml:"default_tags"`
 	MonitoringParams MonitoringParameters `toml:"monitoring_params"`
+	Browse           bool                 `toml:"browse"`
```

Comment:
How about making this a pointer like
```suggestion
	Browse           *bool                `toml:"browse"`
```
as this allows you to decide whether the value was actually _set_ by the user or not.